tools/mksurfdata_map/unit_testers/test_mkgridmapMod.F90

```diff
@@ -334,193 +333,6 @@ subroutine test_gridmap_areaave_srcmask
 
   end subroutine test_gridmap_areaave_srcmask
 
-  !------------------------------------------------------------------------------
```

Comment:
@slevisconsulting here you remove the test subroutine test_gridmap_areaave_srcmask2. However, I think these tests still apply, they just need to be reworked to use gridmap_areaave_srcmask rather than gridmap_areaave. I think after that the tests themselves should still work and pass as before. Is there a different reason you removed this test subroutine?

Reply:
@ekluzek
I agree that the tests would pass if I reworked the calls to use gridmap_areaave_srcmask; however, the test descriptions in `test_gridmap_areaave_srcmask` and `test_gridmap_areaave_srcmask2` are identical, which suggests that repeating the srcmask2 tests as srcmask would be redundant. Pls let me know if I am missing something in my understanding.

Reply:
Ahhh, correct I looked into this to verify. That test list was testing the gridmap_areaoave_srcmask2 interface for gridmap_areaave. Since, that interface was removed there is no reason to have that list of tests in place. So removing it is the right thing to do.